qiskit/pulse/schedule/pulse_schedule.py

```diff
@@ -91,34 +96,42 @@ def name(self) -> str:
     def channels(self) -> ChannelStore:
         return self._channel_store
 
-    def add(self,
-            commands: Union[PulseCommand, List[PulseCommand]],
-            channel: PulseChannel,
-            start_time: int):
-        """Add new pulse command(s) with channel and start time context.
+    def append(self, command: PulseCommand, channel: PulseChannel):
```

Comment:
`append` should just call insert with the final time on the schedule.

Reply:
done